lib/msal-common/src/unifiedCache/entities/AccessTokenEntity.ts

```diff
@@ -34,4 +35,45 @@ export class AccessTokenEntity extends Credential {
 
         return accessTokenKeyArray.join(Separators.CACHE_KEY_SEPARATOR).toLowerCase();
     }
+
+    /**
+     * Create AccessTokenEntity
+     * @param homeAccountId
+     * @param authenticationResult
+     * @param clientId
+     * @param authority
+     */
+    static createAccessTokenEntity(
+        homeAccountId: string,
+        authenticationResult: AuthenticationResult,
+        clientId: string,
+        environment: string
+    ): AccessTokenEntity {
+        const atEntity: AccessTokenEntity = new AccessTokenEntity();
+
+        atEntity.homeAccountId = homeAccountId;
+        atEntity.credentialType = "AccessToken";
+        atEntity.secret = authenticationResult.accessToken;
+
+        const date = new Date();
+        const currentTime = date.getMilliseconds() / 1000;
+        atEntity.cachedAt = currentTime.toString();
+
+        // TODO: Crosscheck the exact conversion UTC
+        // Token expiry time.
+        // This value should be  calculated based on the current UTC time measured locally and the value  expires_in Represented as a string in JSON.
+        atEntity.expiresOn = authenticationResult.expiresOn
```

Comment:
Remember that in the past this was returned as seconds, not milliseconds.

Reply:
yes. Will correct in silent flow logic.